filament/backend/src/vulkan/platform/VulkanPlatformSwapChainImpl.cpp

```diff
@@ -152,10 +161,15 @@ VkResult VulkanPlatformSurfaceSwapChain::create() {
     // Find a suitable surface format.
     FixedCapacityVector<VkSurfaceFormatKHR> const surfaceFormats
             = enumerate(vkGetPhysicalDeviceSurfaceFormatsKHR, mPhysicalDevice, mSurface);
-    FixedCapacityVector<VkFormat> expectedFormats
-            = {VK_FORMAT_R8G8B8A8_UNORM, VK_FORMAT_B8G8R8A8_UNORM};
+    FixedCapacityVector<VkFormat> expectedFormats = {
```

Comment:
why not use a `std::array<>` here? `FixedCapacityVector` does heap allocation which I don't think we should do for 2 elements.

Reply:
done.  good point.